README.md

```diff
@@ -174,9 +174,14 @@ To publish Hibernate Reactive to your local Maven repository, run:
 ### Running tests
 
 To run the tests, you'll need to get the test databases running on your 
-machine. There are three ways to start the test databases. 
+machine. There are three ways to start the test databases.
 
-#### If you have Docker installed
+By default (almost) all of the tests will run with PostgreSQL. To use a different database
```

Comment:
I mean, if you run the build it only runs the test related to PostgreSQL, doesn't need another db running.

Reply:
Yea, I meant "almost all" as in "the other tests will skip". I wouldn't worry too much about this wording because I'll remove it once we can get rid of the DB2 and MySQL specific test classes, which should hopefully be soon.